packages/statistics/index.rst

```diff
@@ -314,40 +314,62 @@ use the :mod:`scipy.stats` sub-module of `SciPy
 Student's t-test: the simplest statistical test
 ------------------------------------------------
 
-1-sample t-test: testing the value of a population mean
+One-sample tests: testing the value of a population mean
 ........................................................
 
 .. image:: two_sided.png
    :scale: 50
    :align: right
 
-:func:`scipy.stats.ttest_1samp` tests if the population mean of data is
-likely to be equal to a given value (technically if observations are
-drawn from a Gaussian distributions of given population mean). It returns
+:func:`scipy.stats.ttest_1samp` tests the null hypothesis that the mean
+of the population underlying the data is equal to a given value. It returns
 the `T statistic <https://en.wikipedia.org/wiki/Student%27s_t-test>`_,
 and the `p-value <https://en.wikipedia.org/wiki/P-value>`_ (see the
 function's help)::
 
     >>> sp.stats.ttest_1samp(data['VIQ'], 0)   # doctest: +ELLIPSIS
     TtestResult(statistic=30.088099970..., pvalue=1.32891964...e-28, df=39)
 
-.. tip::
+The p-value of :math:`10^-28` indicates that such an extreme value of the statistic
+is unlikely to be observed under the null hypothesis. This may be taken as
+evidence that the null hypothesis is false and that the population mean IQ
+(VIQ measure) is not 0.
+
+Technically, the p-value of the t-test is derived under the assumption that
+the means of samples drawn from the population are normally distributed.
+This condition is exactly satisfied when the population itself is normally
+distributed; however, due to the central limit theorem, the condition is
+nearly true for reasonably large samples drawn from populations that follow
+a variety of non-normal distributions.
+
+Nonetheless, if we are concerned that violation of the normality assumptions
+will affect the conclusions of the test, we can use a `Wilcoxon signed-rank test
+<https://en.wikipedia.org/wiki/Wilcoxon_signed-rank_test>`_, which relaxes
+this assumption at the expense of test power::
 
-    With a p-value of 10^-28 we can claim that the population mean for
-    the IQ (VIQ measure) is not 0.
+    >>> sp.stats.wilcoxon(data['VIQ'])  # doctest: +ELLIPSIS
+    WilcoxonResult(statistic=0.0, pvalue=1.8189894...e-12)
 
-2-sample t-test: testing for difference across populations
+Two-sample t-test: testing for difference across populations
 ...........................................................
 
-We have seen above that the mean VIQ in the male and female populations
```

Comment:
We saw that the sample means are different. We want to make an inference about the populations.